config/bano2mimir/default.toml

```
@@ -1,6 +1,15 @@
 nb_threads = 2
 update_templates = true
 
+# If a cosmogony_file is configured, ntfs2mimir will read the admins from this file.
+# If such a file is not configured, ntfs2mimir will read the admins from elasticsearch
+
+# cosmogony_file = "/home/pascal/mimirsbrunn/cosmogony-europe.jsonl.gz"
+
+# configuration for reading the cosmogony file
+french_id_retrocompatibility = true
+langs = [ "fr" ]
```

Maybe I'm a bit late to the party and it's not very important, but wouldn't it be better to have this part in its own cosmogony-related subsection? (and same for other binaries)

```suggestion
  [cosmogony]
  # If a cosmogony_file is configured, ntfs2mimir will read the admins from this file.
  # If such a file is not configured, ntfs2mimir will read the admins from elasticsearch
  # cosmogony_file = "/home/pascal/mimirsbrunn/cosmogony-europe.jsonl.gz"

  # configuration for reading the cosmogony file
  french_id_retrocompatibility = true
  langs = [ "fr" ]
```

---

I've put everything in a [admins] subsection